### PR TITLE
Improve blockcmp

### PR DESCRIPTION
**Pull request by alexeykiselev**

### CODE REVIEW

File: cmd/blockcmp/internal/entities.go [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+func (a *Issue) Equal(b Issue) bool {
+	return a.AssetID == b.AssetID && a.Name == b.Name && a.Description == b.Description &&
+		a.Amount == b.Amount && a.Decimals == b.Decimals && a.Reissuable == b.Reissuable &&
+		a.Script == b.Script && a.Nonce == b.Nonce
+}
+
+func (a *Issue) String() string {
+	return fmt.Sprintf(
```
Comment (nickeskov) [inline cmd/blockcmp/internal/entities.go:79]: I think `reflect` can be helpful here.

File: cmd/blockcmp/internal/entities.go [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+	Description string
+	Amount      int
+	Decimals    int
+	Reissuable  bool
+	Script      string
+	Nonce       int
+}
+
+func (a *Issue) Equal(b Issue) bool {
+	return a.AssetID == b.AssetID && a.Name == b.Name && a.Description == b.Description &&
```
Comment (nickeskov) [inline cmd/blockcmp/internal/entities.go:73]: I think `reflect` can be helpful here.

**Review by nickeskov — CHANGES_REQUESTED** [review]

File: cmd/blockcmp/internal/entities.go (hunk 1 above)
Comment (alexeykiselev) [reply cmd/blockcmp/internal/entities.go:79]: May be later

File: cmd/blockcmp/internal/entities.go (hunk 2 above)
Comment (alexeykiselev) [reply cmd/blockcmp/internal/entities.go:73]: May be later :)

**Review by nickeskov — APPROVED** [review]

**Pull request merged**